src/db/migrate/20200427125343_remove_notifiable_column_from_notifications.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveNotifiableColumnFromNotifications < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :notifications, :notifiable_id, :integer
+  end
+end
```

Comment:
why do we need this migration, there is another one that add it, we'll need to clean up these migrations, please create one migration file for all we need to migrate